### PR TITLE
Cristian Castaneda

### PR DESCRIPTION
**Pull request by crisscast7**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [x] The webpage links to another page (multi-page website): 10 points
- [ ] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [x] The title is centered in front of a background image: 10 points
- [ ] The body images are sized and formatted according to the wire frame: 10 points
- [ ] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [ ] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

Great job with what you completed so far. You will have an opportunity to finish and earn a higher grade. I'll make an announcement in class.

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [x] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [x] Properly commented code: 10 points
- [x] Properly indented code: 10 points

Specific Comments:

**Great effort, Christian! You almost aced it. There are a few issues when you go down to the mobile view, so I would definitely concentrate on media query adjustments during revisions.**

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [x] Responive: 20 points
- [x] Evidence of Effective Use of Design Principles: 20 points
- [x] Links to Resume: 20 points
- [x] Links to at least 3 Projects: 20 points
- [x] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

**Review by crisscast7 — COMMENTED** [review]
changes

File: 02week/nasa/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+      <section id="hero">
+      <h1>NASA</h1>
+      </section>
+      <main>
+<section id="info">
+
+<div class="parent clearfix">
+<div>
+  <h3>Social Media</h3>
```
Comment (demarcogleason) [inline 02week/nasa/index.html]: Good job adding the actual nasa links.... Keep it up big fella!

File: 02week/nasa/index.html [hunk 2]
```diff
@@ -4,10 +4,9 @@
     <title>Intro to Web Development</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
-  <body>
-    
 
-      <section id="hero">
+  <body>
+  <section id="hero">
       <h1>NASA</h1>
```
Comment (ethancrowley) [inline 02week/nasa/index.html]: Nice heading!

**Review by ktee1232 — COMMENTED** [review]
Hey Cristian,

Aesthetically, the checkpoint looks ok but it's definitely missing a few pieces. Keep working at it!

If you want text to show up on top of images, try setting the image as a background in your css.

**Pull request closed without merging**